src/Vimeo.js

```diff
@@ -101,12 +91,12 @@ THE SOFTWARE. */
       }
 
       if(this.options_.poster == "") {
-        $.getJSON(this.baseApiUrl + this.videoId + '.json?callback=?', {format: "json"}, (function(_this){
-          return function(data) {
-            // Set the low resolution first
-            _this.setPoster(data[0].thumbnail_large);
-          };
-        })(this));
+        // $.getJSON(this.baseApiUrl + this.videoId + '.json?callback=?', {format: "json"}, (function(_this){
```

Comment:
We need not comment this out, but just get rid of jQuery.

@polesye can you please suggest a replacement for `$.getJSON()`?